aio/aio-proxy/aio_proxy/search/search_functions.py

```diff
@@ -1,12 +1,20 @@
 from aio_proxy.search.dirigeant import search_dirigeant
-from aio_proxy.search.filters import filter_search
-from aio_proxy.search.helpers import sort_and_execute_search
+from aio_proxy.search.filters import filter_by_siren, filter_search
+from aio_proxy.search.helpers import is_siren, sort_and_execute_search
 from elasticsearch_dsl import query
 
 
 def search_text(index, offset: int, page_size: int, **params):
     query_terms = params["terms"]
     s = index.search()
+
+    # Filter by siren first (if query is a `siren` number, and return search results
+    # directly.
+    if is_siren(query_terms):
```

Comment:
No analysis for other params before ? 
If there is no params, go for it but if there are maybe it is another kind of search (eg for dirigeants for instance). 
My opinion is to be sure that there is only siren in query. If there is other kind of informations (in params). Don't do that.

Reply:
I did discuss this case with @XavierJp yesterday, and we concluded that if a siren number is entered, more often than not no other filters are used.

Reply:
ok let's do it that way then